src-examples/ProxyInterfaceConsumerViaNuGet/Person.cs

```diff
@@ -52,7 +52,7 @@ public void In_Out_Ref1(in int a, out int b, ref int c)
             b = 1;
         }
 
-        public int In_Out_Ref2(in Address a, out Address b, ref Address c)
+        public int In_Out_Ref2(in IAddress a, out Address b, ref IAddress c)
```

Comment:
Not sure why I had to change this

Reply:
I'll take a look as the master branch.

Best is that you revert all changes to this project and just unload it in your solution.